shared/js/bluetooth_helper.js

```diff
@@ -141,6 +144,7 @@ var BluetoothHelper = function() {
     } else {
       _bluetooth.addEventListener('enabled', _getAdapter);
       _bluetooth.addEventListener('adapteradded', _getAdapter);
+      _bluetooth.addEventListener('disabled', _resetAdapter);
```

Comment:
Sure, the `_isReady` property and `resetAdapter` should be private. The helper does not expose them. And should not be exposed them. So that you cannot access them via `subject`.